lua/tests/pickers/oldfiles__escape_tilde.lua

```diff
@@ -0,0 +1,27 @@
+local helper = require "telescope.testharness.helpers"
```

Comment:
yeah we dont need that file, if we already have a automated test. can you remove it.

Thanks :)